pkg/config/protocol.go

```diff
@@ -28,6 +28,8 @@ import "strings"
 type Protocol string
 
 const (
+	// ProtocolAuto enables protocol sniffing.
+	ProtocolAuto Protocol = "Auto"
```

Comment:
I'm not sure how this will be used - do we want the sniffing on ports that user explicitly configures ? 
I assumed this is for ports that don't have an explicit port - if the user needs to specify port=auto, they might as well put the real port.

Reply:
The port protocol configured with `auto` not specified with use protocol sniffing.